fulton-server/src/services/fulton-mongo-entity-service.ts

```diff
@@ -0,0 +1,65 @@
+import { FultonService, Injectable, IUser, QueryResult } from "../index";
+import { MongoRepository } from "typeorm";
+import { IEntityService, Inject, QueryParams } from "../interfaces";
+import { FultonApp } from "../fulton-app";
+
+
+@Injectable()
+export class MongoEntityService<TEntity> implements IEntityService<TEntity> {
```

Comment:
Not a 100% sure on the intention of this class. Can you add a comment? 

I was going to ask if it made sense to call this a Repository. I was expecting anything mongo related to be associated with a repository rather than a service which I am associating with the business layer. Thoughts?